src/core/GameKeyHandler.ttslua

```diff
@@ -134,23 +134,29 @@ function discardObject(playerColor, hoveredObject)
 end
 
 -- actually performs the discarding of the object and tokens / tiles on it
-function performDiscard(playerColor, hoveredObject)
+function performDiscard(playerColor, targetObject)
+  -- destroy temporary chaos tokens instead
```

Comment:
is this going to trigger just for chaos tokens, or temporary action tokens, too?

Reply:
Just for chaos tokens to stop them from entering the chaos bag this way